lib/plugins/statistics.js

```diff
@@ -0,0 +1,23 @@
+module.exports = inject
+
+function inject (bot) {
+  function requestStatistics () {
+    return new Promise(function (resolve, reject) {
+      bot._client.write('client_command', { actionId: 1 })
+      bot._client.once('statistics', parseStatistics)
+
+      function parseStatistics (packet) {
+        const parsedPacket = {}
+
+        packet.entries.forEach(function (statistic) {
+          parsedPacket[statistic.name] = statistic.value
```

Comment:
instead of getting the id's, can we get the actual name of what the statistic is?

Reply:
you can test it, it returns an object with names
you can use like data.deadCount

Reply:
it doesnt, test with 1.16.5